cirq-web/cirq_ts/src/bloch_sphere/components/vector.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 The Cirq Developers
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {ArrowHelper, Vector3} from 'three';
+
+/**
+ * Adds a state vector to the bloch sphere.
+ * @param vectorData information representing the location of the vector.
+ * @returns an ArrowHelper object to be rendered by the scene.
+ */
+interface Vector {
+  x: number;
+  y: number;
+  z: number;
+  v_length: number;
```

Comment:
If this is the vector length, then recommend removing the v_ prefix since it's already the property of a vector object.